portfolio/src/main/java/com/google/sps/servlets/ActivitiesServlet.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/activity-data")
+public class ActivitiesServlet extends HttpServlet {
+
+  private Map<String, Integer> activityVotes = new HashMap<>();
```

Comment:
FYI: This may not be threadsafe so could be inconsistent if multiple users hit the server at once. 

Old article but a good explanation can be found here: https://www.javaworld.com/article/2072798/write-thread-safe-servlets.html

If you can make this threadsafe easily do so. If not, no need to waste time on it this week. Just create an issue to comeback and fix it for your knowledge.

Reply:
I'll make the issue, since I'm accessing the HashMap in both doGet() and doPost()